db/seeds.rb

```diff
@@ -77,6 +77,20 @@ def get_country
     )
 
     puts 'Created default projects'
+
+    opensource = u.courses.create!(
+      title: 'OpenSource',
+      description: 'How to contribute to open source projects.',
+      status: 'Active'
+    )
+
+    gitcourse = u.courses.create!(
```

Comment:
same as above. If we are not using those variables: `gitcourse` and `opensource`, we should consider removing them.

Reply:
Fixed.